pkg/logs/logs.go

```diff
@@ -9,7 +9,10 @@ import (
 	"errors"
 	"fmt"
 	"sync/atomic"
+	"time"
 
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
```

Comment:
I didn't notice on first review, but having to import a package from `cmd/` in `pkg/logs/` is a big code smell.

It looks like it's not the only place in `pkg/` where we import `cmd/`, but still better to avoid this: we're introducing a dependency direction we shouldn't have IMO.

Instead, the `logs` pkg could expose a new exported function that autoconfig would call once it's finished its first run, WDYT?

Reply:
Good catch. I've pushed a commit which should avoid this by using an anonymous function to retrieve the AC instance.